### PR TITLE
Update BriefSubmitted.js

### PR DESCRIPTION
**Pull request by nellied**
Replaced button style with a link, updated copy and removed some break tags.

### CODE REVIEW

File: apps/marketplace/components/Brief/BriefSubmitted.js [hunk 1]
```diff
@@ -20,31 +20,30 @@ const BriefSubmitted = props =>
                   until {format(new Date(props.brief.applicationsClosedAt), 'MMMM Do, YYYY [at] ha [in Canberra]')}
                 </span>}
             </p>
-            <a className="uikit-btn" href={`/${props.brief.frameworkSlug}/opportunities/${props.brief.id}`}>
-              View live brief
-            </a>
+            <a href={`/${props.brief.frameworkSlug}/opportunities/${props.brief.id}`}>View live opportunity</a>
```
Comment (marcoalfonso) [inline apps/marketplace/components/Brief/BriefSubmitted.js]: Why did you remove the className="uikit-btn" ?

Comment (malmas18) [reply apps/marketplace/components/Brief/BriefSubmitted.js]: Neil wanted to save some vertical space.

So the class was removed in favour of a normal link.

#MAR-1429
